test/performance/odp_ipsec.c

```diff
@@ -632,17 +642,22 @@ run_measure_one(ipsec_args_t *cargs,
 		time_record_t *start,
 		time_record_t *end)
 {
+	int in_flight, pkts_allowed, num_out, num_enq, rc = 0;
```

Comment:
I think num_enq is not a very good name here in the sync case which does not really enqueue packets. The async case seems to use num_pkts which could be used here too (and it would have the added benefit of making the two functions more similar in variable naming).

Reply:
No strong opinion on variable name. Will rename as part of v4.